ToroHealth_client.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
add this and the server launch file to the .gitignore to avoid checking them in.

Reply:
Never mind, I am merging as is.  There are some bugs which I think were there from the initial work I did for 1.15.1.  I will try to fix those over the next few days, I think you got me past the part I was stuck on.

Reply:
What kind of bugs do you mean?

Reply:
I only looked at it for a few minutes, but I noticed the in world bars weren't turning off with the settings.  The in world bars were jittery, like they where not using the partial tick to set their position.  The timing on the bar drain is too slow, and the number stays much longer than it should.  I had some uncommitted changes that might help.  I remember being stuck on the rendering changes in 1.15, but it looks like you got through all of them with no issue.